### PR TITLE
Add social media

### PR DESCRIPTION
**Pull request by alexandercyu**
- Added social media icons to the right side of the menu in anticipation of adding social media presence.   These are the ones mentioned in the meeting on 2024-01-04.
- Added CSS to change the appearance of the social media icons from the default blue text with underline.
(Only applied this to some of the icons so the difference in appearance can be compared)

### CODE REVIEW

File: templates/partials/menu.html [hunk 1]
```diff
@@ -32,6 +32,13 @@
                     </li>
                 </ul>
             </div>
+            <div class="col-1">
+                <a class="social-media-icon" href=""><i class="fab fa-linkedin"></i></a>
+                <a class="social-media-icon" href="" target="blank">&#120143;</a>
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply templates/partials/menu.html:37]: They're still available through a CDN. Kit links are not required:

https://cdnjs.com/libraries/font-awesome

File: templates/partials/menu.html [hunk 2]
```diff
@@ -32,6 +32,13 @@
                     </li>
                 </ul>
             </div>
+            <div class="col-1">
+                <a class="social-media-icon" href=""><i class="fab fa-linkedin"></i></a>
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply templates/partials/menu.html:36]: I see... I think we want them all to have the same class either way and then we can apply logo colour changes in the css class itself.

The blue doesn't match the colour pallette of the rest of the page. We should either choose a colour that matches or keep these grey.

File: templates/partials/menu.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexandercyu) [reply templates/partials/menu.html:37]: Thanks.  I don't think a CDN was mentioned on the fontawesome site.  Everything I clicked linked back to getting a kit link.

File: templates/partials/menu.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexandercyu) [reply templates/partials/menu.html:36]: Added target="blank" to all icons.

Added css class to all icons.

Changed icons to large size version (icons looked too small to me).

File: templates/partials/menu.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexandercyu) [reply templates/partials/menu.html]: Switched to fontawesome 6.5.1.

Note: The github icon in the remote processing page looks smaller for some reason.  Both the github and documentation icons are using the large version of the icon.

File: static/css/daccs-css.css [hunk 3]
```diff
@@ -103,6 +103,12 @@
     width: 40%;
 }
 
+.social-media-icon{
+    color: inherit;
+    text-decoration: none;
+
```
[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/daccs-css.css]: Removed

File: templates/partials/menu.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexandercyu) [reply templates/partials/menu.html]: Put class and target="blank" on all icon links.

File: static/css/daccs-css.css [hunk 4]
```diff
@@ -103,6 +103,12 @@
     width: 40%;
 }
 
+.social-media-icon{
+    color: inherit;
+    text-decoration: none;
```
[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/daccs-css.css]: Switched to fontawesome logo

File: templates/partials/menu.html [hunk 5]
```diff
@@ -32,6 +32,13 @@
                     </li>
                 </ul>
             </div>
+            <div class="col-1">
+                <a class="social-media-icon" href="" target="blank"><i class="fa-brands fa-linkedin fa-lg"></i></a>
```
Comment (mishaschwartz) [inline templates/partials/menu.html:36]: `target="_blank"`

Comment (alexandercyu) [reply templates/partials/menu.html]: Fixed

**Review by mishaschwartz — APPROVED** [review]
Let's hold off on pulling this in until we have the href values as well. I'm going to try to set up as many of the social media pages this week and next.

**Comment by mishaschwartz** [comment]
Added most of the social links. We still need to figure out how to host/pay for discourse before we update that link as well.

**Pull request merged**